ownCloud/Tools/VendorServices.swift

```diff
@@ -45,6 +46,18 @@ class VendorServices : NSObject {
 		return ""
 	}
 
+	var isBetaBuild: Bool {
+		if let isBetaBuild = self.classSetting(forOCClassSettingsKey: .isBetaBuild) as? Bool {
+			if isBetaBuild {
```

Comment:
@pablocarmu I think you could just `return isBetaBuild` here, no if condition needed